### PR TITLE
[MNG-8301] Can't build a project without modelVersion=4.1.0 but with 4.1.0 (beta-4) XSD

### PR DESCRIPTION
**Pull request by mthmulders**

### CODE REVIEW

**Comment by mthmulders** [comment]
Note that the above pipeline succeeds, because it runs with Maven 3.9.7. When run with 4.0.0-beta-5-SNAPSHOT, it fails.

File: core-it-suite/src/test/resources/mng-8301-project-without-modelversion/module-a/pom.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.1.0 https://maven.apache.org/xsd/maven-4.1.0-beta-4.xsd">
```
Comment (DidierLoiseau) [inline core-it-suite/src/test/resources/mng-8301-project-without-modelversion/module-a/pom.xml:2]: `xmlns` does not match `schemaLocation`
```suggestion
<project xmlns="http://maven.apache.org/POM/4.1.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.1.0 https://maven.apache.org/xsd/maven-4.1.0-beta-4.xsd">
```

File: core-it-suite/src/test/resources/mng-8301-project-without-modelversion/module-b/pom.xml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.1.0 https://maven.apache.org/xsd/maven-4.1.0-beta-4.xsd">
```
Comment (DidierLoiseau) [inline core-it-suite/src/test/resources/mng-8301-project-without-modelversion/module-b/pom.xml:2]:
```suggestion
<project xmlns="http://maven.apache.org/POM/4.1.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.1.0 https://maven.apache.org/xsd/maven-4.1.0-beta-4.xsd">
```

File: core-it-suite/src/test/resources/mng-8301-project-without-modelversion/module-b/pom.xml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  <parent>
+    <groupId>org.apache.maven.its.mng8301</groupId>
+    <artifactId>m4womv</artifactId>
+  </parent>
+
+  <artifactId>module-b</artifactId>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.apache.maven.repro</groupId>
```
Comment (DidierLoiseau) [inline core-it-suite/src/test/resources/mng-8301-project-without-modelversion/module-b/pom.xml:13]: It’s the wrong `groupId`
```suggestion
      <groupId>org.apache.maven.its.mng8301</groupId>
```

File: core-it-suite/src/test/resources/mng-8301-project-without-modelversion/pom.xml [hunk 4]
```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.1.0 https://maven.apache.org/xsd/maven-4.1.0-beta-4.xsd">
```
Comment (DidierLoiseau) [inline core-it-suite/src/test/resources/mng-8301-project-without-modelversion/pom.xml:2]:
```suggestion
<project xmlns="http://maven.apache.org/POM/4.1.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.1.0 https://maven.apache.org/xsd/maven-4.1.0-beta-4.xsd">
```

**Review by DidierLoiseau — COMMENTED** [review]
I noticed some issues. I think you should use the 4.1.0 if you want to benefit from optional versions?

At least, if I change that, it works.

**Pull request closed without merging**